pkg/util/util.go

```diff
@@ -0,0 +1,37 @@
+package util
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+// MegaByte denotes one megabyte and is intended to be used as a limit in `ReadAtMost`.
+const MegaByte = 1 << 20
```

Comment:
Should/can this be a conf file entry somewhere?

Reply:
No, users shouldn’t ever end up having to make a choice.